Source/Server/Core/Master.cs

```diff
@@ -66,6 +66,7 @@ public static void Main()
 
```

Comment:
Avoids keeping the thread alive permanently. Task Delay will provide a less blocking method and personally, I find it to be cleaner.


public static async Task Main()
{
    Threader.GenerateServerThread(Threader.ServerMode.Start);
    Threader.GenerateServerThread(Threader.ServerMode.Console);
    Threader.GenerateServerThread(Threader.ServerMode.Discord);

    await Task.Delay(-1); // Keeps the main method alive without blocking the thread
}